docs/api/deletion_tab.md

```diff
@@ -0,0 +1,53 @@
+# Deletion
```

Comment:
Can you change it to DeletionTab please ? :)

Reply:
done